src/ducks/balance/GroupPanel/GroupPanel.jsx

```diff
@@ -98,4 +98,4 @@ GroupPanel.defaultProps = {
   withBalance: true
 }
 
-export default React.memo(GroupPanel)
+export default GroupPanel
```

Comment:
This is because `group` was updated but `GroupPanelSummary` was not re-render?

Reply:
The GroupPanel was never re-rendered even if the group changed. The rows are updated by side-effect because their props triggers change. This gives the impression that everything is fine except for the summary.

<img src="https://github.com/cozy/cozy-banks/assets/7434420/c4a2d352-ff31-4d48-96a9-a9dfae7598b3" height="400px" />

![Capture d’écran 2024-02-05 à 16 42 35](https://github.com/cozy/cozy-banks/assets/7434420/2544a7a0-c60b-427e-8eec-6d2c43e73477)

Reply:
The graph is super expensive. I tried to avoid render re-renders by splitting component, but I didn't manage to optimise that part. You'd probably have to use useQuery more than connects to avoid them rendering the whole tree for each query result.